### PR TITLE
Update markdown for docusaurus v3.

### PR DESCRIPTION
**Pull request by gz**

### CODE REVIEW

File: docs/demo/demo.md [hunk 1]
```diff
@@ -122,7 +120,6 @@ To see a visualization of the relationships between tables and views,
 expand the following, which shows table names in lowercase and view
 names in uppercase:
 
-<details><summary>E-R diagram of tables and views</summary>
```
Comment (mihaibudiu) [inline docs/demo/demo.md:125]: so this text is lost?

File: docs/docker.md [hunk 2]
```diff
@@ -82,7 +82,7 @@ You also need `curl` and a web browser such as Chrome or Firefox.
    interface to become available.  On fast systems, this includes a
    pause of a few seconds in which nothing is logged.
 
-3. Visit <http://localhost:8080/> in your web browser to bring up the
+3. Visit `http://localhost:8080/` in your web browser to bring up the
```
Comment (mihaibudiu) [inline docs/docker.md:85]: This is not longer the same markdown, it used to be a link

**Review by mihaibudiu — APPROVED** [review]

File: docs/demo/demo.md (hunk 1 above)
Comment (gz) [reply docs/demo/demo.md:125]: I don't know how to support this line in the new format, but I don't think it's very important

File: docs/docker.md (hunk 2 above)
Comment (gz) [reply docs/docker.md:85]: It is a link in new markdown.

**Pull request merged**